manual/commands.md

```diff
@@ -0,0 +1,633 @@
+# Help
+Once installed, if you run the `meza` command all by itself, it will display *help* like the following:
+```bash
+Mediawiki EZ Admin
+
+Usage: meza COMMAND [directives]
```

Comment:
Some meza commands below use sudo and others do not. Is this intentional?

Reply:
Maybe here or in some other doc like the main readme it would help to explain which user can/should be used to install and run meza. Should root be used or should some other non-root sudoer be used? Does it matter?